scripts/portfolio.js

```diff
@@ -0,0 +1,50 @@
+(function(module) {
+
+  function Project (opts) {
+    this.author = opts.author;
+    this.title = opts.title;
+    this.category = opts.category;
+    this.authorUrl = opts.authorUrl;
+    this.publishedOn = opts.publishedOn;
+    this.body = opts.body;
```

Comment:
I'm also in the process of adding a new property to my object constructor. I want to add in a small screenshot of each project I have, and plan to update my current template as well.